doc-content/enterprise-only/openshift/environment-immutable-server-kjar-proc.adoc

```diff
@@ -0,0 +1,82 @@
+** *KIE Server Certificate Name* (`KIE_SERVER_HTTPS_NAME`): The name of the certificate in the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Keystore Password* (`KIE_SERVER_HTTPS_PASSWORD`): The password for the keystore that you created in <<secrets-kie-create-proc>>.
+** *KIE Server Container Deployment* (`KIE_SERVER_CONTAINER_DEPLOYMENT`): The identifying information of the decision services (KJAR files) that the deployment must pull from the Maven repository. The format is: `<containerId>=<groupId>:<artifactId>:<version>`. You can provide two or more KJAR files using the `|` separator, for example: `containerId=groupId:artifactId:version|c2=g2:a2:v2`.
+** *Maven repository URL* (`MAVEN_REPO_URL`): The URL for the Maven repository.
+** *Maven repository username* (`MAVEN_REPO_USERNAME`): The username for the Maven repository.
+** *Maven repository password* (`MAVEN_REPO_PASSWORD`): The password for the Maven repository.
+** *Disable KIE server management* (`KIE_SERVER_MGMT_DISABLED`): You must set this parameter to `true` for an immutable deployment.
+** *KIE Server Startup Strategy* (`KIE_SERVER_STARTUP_STRATEGY`): You must set this parameter to `LocalContainersStartupStrategy` for an immutable deployment.
+** *ImageStream Namespace* (`IMAGE_STREAM_NAMESPACE`): The namespace where the image streams are available. If the image streams were already available in your OpenShift environment (see <<imagestreams-file-install-proc>>), the namespace is `openshift`. If you have installed the image streams file, the namespace is the name of the OpenShift project.
+ifdef::PAM[]
++
+. If you want to use {LOCAL_CENTRAL} or Smart Router, set some of the parameters that were displayed in the sample command line after you deployed the monitoring infrastructure (see <<environment-immutable-monitoring-proc>>), namely:
+** Set `KIE_ADMIN_USER`, `KIE_ADMIN_PWD`, `KIE_SERVER_USER`, `KIE_SERVER_PWD`, and `KIE_SERVER_ROUTER_SERVICE` to the values that were displayed for these parameters.
+** Set `KIE_SERVER_CONTROLLER_USER` to the value that was displayed for `KIE_SERVER_MONITOR_USER`.
+** Set `KIE_SERVER_CONTROLLER_PWD` to the value that was displayed for `KIE_SERVER_MONITOR_PWD`.
+** Set `KIE_SERVER_CONTROLLER_SERVICE` to the value that was displayed for `KIE_SERVER_MONITOR_SERVICE`.
+endif::PAM[]
++
+include::ldap_rhsso.adoc[]
+
+ifdef::PAM[]
+include::externaldb-params.adoc[]
+endif::PAM[]
+. Complete the creation of the environment, depending on the method that you are using:
```

Comment:
This reads a little weird. You qualify the first option (In OpenShift Web UI) so maybe qualify the second as well (If you are using BLAH, complete.. or In BLAH, complete..

Reply:
To be reviewed with procedure reorg. Not changing now as this needs to be the same in all similar procedures.